README.md

```diff
@@ -85,10 +85,10 @@ ollama:
 ollama:
   models:
     - llama2
-  
+
 ingress:
   enabled: true
-    hosts:
+  hosts:
```

Comment:
This is the fix.

The rest of the changes are just vscode stripping trailing whitespace.